package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "hexo-filter-nofollow",
+  "version": "2.0.0",
```

Comment:
`1.0.0` or less than `1.0.0` version?

Reply:
https://www.npmjs.com/package/hexo-filter-nofollow/v/1.0.1 is already published (which doesn't include this PR).